piksi_multi_rtk_ros/launch/piksi_multi_base_station.launch

```diff
@@ -2,13 +2,28 @@
 
 <launch>
 
-  <node pkg="piksi_multi_rtk_ros" type="piksi_multi.py" name="piksi" output="screen" respawn="true">
+  <!-- Settings: will overwrite default setting loaded form file.  -->
+  <!-- Available interfaces: serial or tcp. -->
+  <arg name="interface"                   default="tcp"/>
+  <!-- If interface is serial, this specifies baud rate. -->
+  <arg name="baud_rate"                   default="230400"/>
+  <!-- If interface is tcp, this specifies the address. -->
```

Comment:
Maybe specify which address. Should be the one of the computer on which the base station launch file is running, correct?

Reply:
Nope, it's the address of Piksi itself. I'll add a comment about it